### ISSUE TITLE
Autopilot Level Off Sound

### ISSUE THREAD

**Issue by OFC1983** [flybywiresim/a32nx#6503]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2022-01-07T19:01:38+00:00",
  "ref": "master",
  "sha": "8a0ce720e05c944b66ca59253d29f852eed0db38",
  "actor": "crocket63",
  "event_name": "manual"
}
```

### Describe the bug

When performing a manual descend from the Altitude selected in the FCU the Autopilot Level Off sound continuously repeats itself without ever stopping.

### Expected behavior

The sound should only alarm once and then automatically stop.

### Steps to reproduce

Select an altitude on the FCU Panel for the autopilot and climb to this altitude. Once reaching the altitude start your descend and listen to the continuous sound. I can only stop it by selecting a different altitude in the FCU. This is a bit annoying especially during the approach phase when working with multiple altitude changes and manual flying.

**Comment by beheh** [comment]
The conditions will be cleaned up and fixed with the new FWC.

**Comment by tracernz** [comment]
That’s the c-chord warning you that you’re deviating from the FCU altitude. You can cancel it by pressing the master warning push button.

**Comment by OFC1983** [comment]
> That’s the c-chord warning you that you’re deviating from the FCU altitude. You can cancel it by pressing the master warning push button.

Thank you for the tip, fixing it with a workaround is easy, but a workaround should not be necessary. But its comforting to know that this will be cleaned up with the new FWC.

**Comment by tracernz** [comment]
> > That’s the c-chord warning you that you’re deviating from the FCU altitude. You can cancel it by pressing the master warning push button.
> 
> Thank you for the tip, fixing it with a workaround is easy, but a workaround should not be necessary. But its comforting to know that this will be cleaned up with the new FWC.

The logic sounds like it's working properly to me? You were at the FCU altitude, then began descending without changing the FCU altitude? That satisfies the conditions for a continuous C-chord: https://github.com/flybywiresim/a32nx/pull/3281

**Comment by OFC1983** [comment]
> > > That’s the c-chord warning you that you’re deviating from the FCU altitude. You can cancel it by pressing the master warning push button.
> > 
> > 
> > Thank you for the tip, fixing it with a workaround is easy, but a workaround should not be necessary. But its comforting to know that this will be cleaned up with the new FWC.
> 
> The logic sounds like it's working properly to me? You were at the FCU altitude, then began descending without changing the FCU altitude? That satisfies the conditions for a continuous C-chord: #3281

You seem to be correct. However, there is one exception that the logic currently does not take into consideration:

![image](https://user-images.githubusercontent.com/72392200/148624260-2085a741-6b26-4c7b-b69f-095e022b8c79.png)
 It states that the altitude alert is inhibted in approach when capturing the glide slope. However, I often found that the alarm sounds when performing the manual approach. Not sure if this also applies to the Autopilot approach...

**Comment by tracernz** [comment]
Good point. 👍

**Comment by beheh** [comment]
Yes, there are a few subtle conditions like this that can cause the warning to be inhibited or reset, which we don't faithfully simulate (yet).
